packages/client/src/stores/ReportStore.ts

```diff
@@ -217,6 +239,8 @@ export default class ReportStore {
   @action.bound
   handleCanaryRunSelection(canaryExecutionResult: CanaryExecutionResult) {
     this.selectedCanaryExecutionResult = canaryExecutionResult;
+    this.canaryResult = canaryExecutionResult.result;
     this.displayMetricOverview = true;
+    this.metricSetPairListId = canaryExecutionResult.metricSetPairListId as string;
```

Comment:
Why do you have to cast this to a `string` if the type is already of type `string`?
https://github.com/Nike-Inc/referee/blob/master/packages/client/src/domain/Kayenta.ts#L255